--- sslh-1.10/sslh-main.c~	2011-11-26 22:16:43.000000000 +0100
+++ sslh-1.10/sslh-main.c	2012-02-17 22:56:31.899567806 +0100
@@ -177,6 +177,8 @@
 
    setup_signals();
 
+   write_pid_file(pid_file);
+
    drop_privileges(user_name);
 
    /* New session -- become group leader */
@@ -185,8 +187,6 @@
        CHECK_RES_DIE(res, "setsid: already process leader");
    }
 
-   write_pid_file(pid_file);
-
    /* Open syslog connection */
    setup_syslog(argv[0]);
 
